virtcontainers/pkg/cloud-hypervisor/client/docs/DiskConfig.md

```diff
@@ -5,9 +5,13 @@
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
 **Path** | **string** |  | 
+**Readonly** | **bool** |  | [optional] [default to true]
```

Comment:
I thought we agreed this shouldn't default to true.

Reply:
@sboeuf I believe the `default to true` comes from the `cloud-hypervsior.yaml` [here](https://github.com/sameo/cloud-hypervisor-1/blob/76dc26b0d1715b254680cd715273ab682ef437ed/vmm/src/api/openapi/cloud-hypervisor.yaml#L279). Note that all the code under `client/` here are auto-generated based on the yaml file.

Reply:
Yes this needs to be fixed from CH directly.

Reply:
My bad, this is fixed now. @likebreath You probaly want to fix accordingly.